src/FortifyServiceProvider.php

```diff
@@ -73,6 +75,10 @@ public function register()
         $this->app->bind(StatefulGuard::class, function () {
             return Auth::guard(config('fortify.guard', null));
         });
+
+        $this->app->bind(PasswordBroker::class, function () {
```

Comment:
Is this change totally necessary? I'm worried about overriding this binding for existing applications.

Reply:
I reverted the password broker changes.